column_buffer_amd64.go

```diff
@@ -33,7 +33,7 @@ func writeValuesBitpackAVX2(values unsafe.Pointer, rows array, size, offset uint
 //go:noescape
 func writeValues32bitsAVX2(values unsafe.Pointer, rows array, size, offset uintptr)
 
-//go:noescpae
+//go:noescape
```

Comment:
This was a bug which caused local variables to escape to the heap since the compiler could not determine whether the function would retain pointers that were passed as arguments.

It was surprising that neither `go build` nor `go vet` picked it up 🤷 I was hoping they would warn about invalid compiler directives.